### PR TITLE
In EditScreen, make ClearButton look the same as Figma

### PR DESCRIPTION
**Pull request by kosenda**
## Issue
none

## Overview (Required)
- I fixed the ClearButton in EditScreen because it looked different from Figma.
- In Figma, the shadows were blurred, but since it had little effect on the appearance and it seemed impossible to achieve this without a certain amount of ingenuity, a fill was used.

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After | Figma
:--: | :--: | :--:
 |   | 

## Figma
https://www.figma.com/design/XUk8WMbKCeIdWD5cz9P9JC/DroidKaigi-2024-App-UI?node-id=55588-34615&t=VpbDVe3CvbfpqvxU-4

### CODE REVIEW

File: feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt [hunk 1]
```diff
@@ -554,8 +556,15 @@ private fun ImagePickerWithError(
                     onClick = onClearImage,
                     modifier = Modifier
```
Comment (takahirom) [inline feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt:557]: I'm not sure, but it seems that there is a shadow modifier. I think you already know that, but if you don't, could you check it out?

Comment (kosenda) [reply feature/profilecard/src/commonMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardScreen.kt:557]: I tried shadow and thought it was a little off from Figma, but after your comment I tried again and got almost the same look, so I fixed it 🙇

**Comment by kosenda** [comment]
Here is a comparison of before and after modification of your comment

Before | After | Figma
:--: | :--: | :--:
  |   |

**Comment by takahirom** [comment]
Thank you for your meticulous work and addressing this issue!

**Review by takahirom — APPROVED** [review]

**Pull request merged**